server/app.js

```diff
@@ -51,15 +47,6 @@ if (process.env.NODE_ENV === 'development') {
 /** favicon setup */
 app.use(favicon(path.join(__dirname, '../public/img/favicon.ico')))
 
-/** security setup */
-app.use(
-  helmet({
```

Comment:
should we leave this in? We discussed either removing it or keeping it.

Reply:
IMO we should keep this, it fills the same role that Rails handles as part of the framework: https://guides.rubyonrails.org/v4.1/security.html#default-headers

Reply:
[b0ab09e](https://github.com/AMP-SCZ/dpdash/pull/518/commits/b0ab09ecac96b308e778e45f782c00d3d1fc2f6a)